src/utils/typography.js

```diff
@@ -3,15 +3,18 @@ import CodePlugin from 'typography-plugin-code';
 import { MOBILE_MEDIA_QUERY } from 'typography-breakpoint-constants';
 
 const options = {
-  baseFontSize: '18px',
+  baseFontSize: '14px',
   baseLineHeight: 1.45,
   scaleRatio: 2.25,
+  googleFonts: [name:'Lato'],
```

Comment:
This isn't valid javaScript, and with it in place, it breaks the whole site. 

Please make the changes required and test that it works before amending this commit